test/mocks/application/DocumentRootInMemoryRepository.scala

```diff
@@ -172,4 +173,6 @@ class DocumentRootInMemoryRepository extends GeneralApplicationRepository {
   def removeProgressStatuses(appId: String, progressStatuses: List[ProgressStatuses.ProgressStatus]): Future[Unit] = ???
 
   def getOnlineTestApplication(appId: String): Future[Option[OnlineTestApplication]] = ???
+
+  def progressCheck(applicationId: String, progressStatus: ProgressStatus): Future[Boolean] = ???
```

Comment:
I might just delete it - it was when I was thinking we needed to check the progress status when actually it's totally redundant in this case.